web/cypress/e2e/rest/pluginTemplate-create-edit-delete-plugin-template.cy.js

```diff
@@ -48,7 +48,9 @@ context('Create Configure and Delete PluginTemplate', () => {
     cy.visit('/');
     cy.contains('Route').click();
     cy.get(selector.empty).should('be.visible');
-    cy.contains('Advanced').should('be.visible').click();
+    cy.contains('Advanced').trigger('mouseover');
+    cy.wait(50);
```

Comment:
We should avoid abusing wait, and use some should assertions to ensure that the tests are stable.